long_tests/test_deep_training.py

```diff
@@ -195,3 +197,22 @@ def get_mmdet_config():
     dict(type='Collect', keys=['img', 'gt_bboxes', 'gt_labels'])
 ]
 """
+
+def calculate_iou(bounding_box: resources_pb2.BoundingBox, expected_coords: List[float]):
```

Comment:
You could express the type more precisely as a tuple of 4 floats. 

`Tuple[float, float, float, float]`

Reply:
Thanks